repository/v.json

```diff
@@ -393,6 +393,18 @@
 				}
 			]
 		},
+		{
+			"name": "Vintage Relative Lines",
+			"details": "https://github.com/bicarlsen/vintage_relnums",
+			"issues": "https://github.com/bicarlsen/vintage_relnums/issues",
```

Comment:
Because your issues are in the default location, please don't link to them here (ie. remove this line)

Reply:
Thank you for the input. I'll make the change.